core/modules/node/node.module

```diff
@@ -1863,6 +1865,10 @@ function node_block_info() {
     'description' => t('Displays a node in a block.'),
     'class' => 'NodeBlock',
   );
+  $blocks['empty'] = array(
+    'info' => t('Create content'),
+    'description' => t('A prompt to create some content with appropriate links.'),
+  );
```

Comment:
> There would be zero usefulness of manually placing this block after the initial installation.

That's not true, this is the most popular block in my admin dashboard module. And for almost every site I built with panels, I hand-coded one of these blocks so that administrators would have a list of all the content they could create.  I'd call this an 80% use-case feature! (Though, admittedly not necessarily related to this issue).